fala/assets-src/sass/main.scss

```diff
@@ -215,7 +215,29 @@ body.fala-dev {
   }
 
   &__grey-box {
-    background-color: govuk-colour("light-grey"); 
+    background-color: govuk-colour("light-grey");
     padding: 1em;
   }
 }
+
+.miles-away {
+  color: $govuk-secondary-text-colour;
+}
+
+.results-list-item {
+  margin-top: govuk-spacing(4);
+  border-top: 1px solid #b1b4b6;
```

Comment:
Is `#b1b4b6` not the same colour as `govuk-colour("light-grey")`

Reply:
It would be the same as `govuk-colour("mid-grey")` (see image below, [here is the link](https://design-system.service.gov.uk/styles/colour/))


<img width="466" alt="Screenshot 2024-04-23 at 09 02 45" src="https://github.com/ministryofjustice/fala/assets/76905544/81436e22-aa7b-4a83-a9f5-cd161ff0b7fd">